webapp/graphite/tags/redis.py

```diff
@@ -151,29 +156,44 @@ def get_series(self, path, requestContext=None):
 
     return TaggedSeries(tags['name'], tags)
 
-  def list_tags(self, tagFilter=None, requestContext=None):
-    return sorted([
+  def list_tags(self, filter=None, limit=None, requestContext=None):
+    results = sorted([
```

Comment:
that's just a detail:
but wouldn't it be faster to first sort the list of tag strings and then build a list of `{'tag': tag}` dicts based on it, instead of having to build a lambda function and then call it many times inside the sort function to extract that tag back out again?

Reply:
furthermore, then the limit could be applied before build the list of dicts